src/login/logind-session.c

```diff
@@ -656,6 +657,14 @@ static int session_start_scope(Session *s, sd_bus_message *properties, sd_bus_er
 
                 description = strjoina("Session ", s->id, " of User ", s->user->user_record->user_name);
 
+                after = strv_new("systemd-logind.service",
```

Comment:
why switch to strv_new()? STRV_MAKE should be just fine?

Reply:
indeed

Reply:
Hmm. the point to use `strv_new()` is to make use of `STRV_IGNORE` which is not possible otherwise. Or am I missing something ?

Reply:
add a brief comment about that please